ui/src/templates/components/TemplateCard.tsx

```diff
@@ -128,7 +128,7 @@ class TemplateCard extends PureComponent<Props & WithRouterProps> {
 
   private handleExport = () => {
     const {router, template, org} = this.props
-    router.push(`orgs/${org.id}/templates/${template.id}/export`)
+    router.push(`/orgs/${org.id}/templates/${template.id}/view`)
```

Comment:
should this be a part of the handleExport function? this is not dealing with exporting anymore? perhaps this function just requires a name change? it seems like this would be more of a handleNameClick concern.

Reply:
Yup, forgot to update the name for handler